mod.config

```diff
@@ -68,8 +53,27 @@ PACKAGING_FAQ_URL="http://wiki.openra.net/FAQ"
 #  - Windows "Add/Remove Programs" list
 PACKAGING_AUTHORS="Example Mod authors"
 
+# Space delimited list of dll files compiled by the mod, which
+# should be copied from the bin directory into your installers
+PACKAGING_COPY_MOD_BINARIES="OpenRA.Mods.Example.dll"
+
+# If your mod depends on OpenRA.Mods.Cnc.dll from the engine set
+# this to "True" to package the dll in your installers.
+# Accepts values "True" or "False".
+PACKAGING_COPY_CNC_DLL="False"
+
+# If your mod depends on OpenRA.Mods.D2k.dll from the engine set
+# this to "True" to package the dll in your installers.
+# Accepts values "True" or "False".
+PACKAGING_COPY_D2K_DLL="False"
```

Comment:
This is odd. Why did you go boolean here?

Reply:
I don't understand the question. How would you have done this? Bearing in mind the limitations of `dotnet package` that we have to work with.

Reply:
`PACKAGING_COPY_ENGINE_FILES` did this before and a list seemed to make more sense. I don't know about the limitations of `dotnet package` though nor why it is used to copy files.

Reply:
With .NET core we can no longer treat assemblies like plain files: publishing binaries for release is a different toolchain operation to compiling development binaries that does additional steps like compile a native launcher and strip unused symbols/assemblies. Publishing the engine project will copy these extra dlls whether we like it or not, so we have to then follow up by deleting them afterwards.